src/components/AttachmentModal.js

```diff
@@ -60,7 +60,7 @@ const propTypes = {
 };
 
 const defaultProps = {
-    sourceURL: null,
+    source: null,
```

Comment:
Is it a bit odd that we defaulted to `null` even though this isn't an allowed prop type? I know this is what we did in the past even, but I wonder if defaulting to `''` would break anything?